db/migrate/20150908120244_add_image_column_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageColumnToPieces < ActiveRecord::Migration
+  def change
+  	add_column :pieces, :image, :string 
```

Comment:
This is one way to do this and it isn't wrong and has advantages. However, I would suggest not storing the image in the database.  The image is associated with a piece type, we already know what image should be associated with say a Rook because we have the type and you already have the color in the database as well.  You can use these two facts to figure out the image at code execution time.